CHANGELOG

```diff
@@ -23,20 +23,19 @@ Version 4.3.3 (development)
 - Palettes can now also be specified with explicit alpha channels. See the
   example file share/palettes-variable-opacity.txt.
 
-- Added loading of data collections (VisIt, Sidre, FMS and Conduit) through
-  parameters '-visit', '-sidre', '-fms' and '-conduit' respectively. The name
-  of the field can be specified through '-g' or '-q' for a quadrature field.
-  Optionally, the cycle and protocol can be provided through 'dc-cycle' and
-  'dc-prot' parameters respectively. Note GLVis must be compiled with MFEM
-  configured with the associated libraries.
-
-- Fixed a bug where discrete textures would not repeat.
+- Added loading of MFEM data collections (VisIt, Sidre, FMS and Conduit) through
+  command line arguments '-visit', '-sidre', '-fms' and '-conduit' respectively.
+  The field to plot can be specified using '-g' or '-q' for a quadrature field.
+  Optionally, the cycle and protocol can be provided through the 'dc-cycle' and
+  'dc-prot' parameters. Note that GLVis must be build using MFEM configured with
+  the associated libraries.
 
 - Added support for 1D/2D data (mesh and grid function) using scalar finite
-  elements with up to 3 vector dimensions
+  elements with up to 3 vector dimensions.
 
-- Added support for 1D/2D vector finite elements with 3 range dimensions
-  (requires MFEM v4.8).
+- Added support for 1D/2D vector finite elements with 3 range dimensions.
```

Comment:
@mlstowell and @najlkin -- can we improve this `CHANGELOG` entry? As it is, it may not be clear what is the difference with the previous one...

Reply:
The previous one says scalar elements with 3 vector dimensions. This says vector elements with 3 range dimensions, so they are different to me 🤔 But it is true it is more technical than most of the items around here, but I do not know how to rewrite it keeping the difference clear.

Reply:
ok, lets leave it as is